tox.ini

```diff
@@ -69,12 +69,15 @@ commands =
 allowlist_externals = sh
 
 [testenv:docs]
-description = docs
+description = sphinx docs and man pages
 deps =
+    -r docs/requirements.txt
     # TODO: switch to a release tag when it's cut
     # We need https://github.com/click-contrib/click-man/pull/62 to handle hidden options
     git+https://github.com/click-contrib/click-man@24ec8377e3c24378417f6fc4f571b4f585d7df6b
 commands =
+    # add "-n" once all errors are fixed
```

Comment:
Which errors?

Reply:
Nit pick mode was complaining about missing references. I fixed them by adding intersphinx, explicit nickpick ignores, and changing the code a bit. For example I had to make the Pydantic submoduels public, so they show up in the docs and can be referenced.

Reply:
Thanks!